modules/collectd/manifests/plugin/docker.pp

```diff
@@ -31,16 +31,21 @@
   # seems to be fixed in puppet 4: https://tickets.puppetlabs.com/browse/PUP-1073
   exec { 'pip install docker':
     path    => ['/usr/local/bin', '/usr/bin', '/bin'],
-    command => 'pip install "docker<=2.6.1"',
+    command => 'pip install "docker==2.7.0"',
```

Comment:
I remember looking at this a while back. What's the current issue with version prior to 2.7?

Reply:
dockerplugin can import some dependencies: https://github.com/alphagov/docker-collectd-plugin/commit/e56ddb84536065786e0a55143faa8c6fc035c119 . 

Strangely it is only on the mirrorer machines that it behaves this way.

Reply:
Which dependencies can't it import? This sounds very similar to the issue Aga and I were trying to address with the previous round of changes, but it was a while ago, so I'm struggling to remember all the details...

Reply:
21709:Jun  3 14:07:03 ip-10-12-6-151.eu-west-1.compute.internal collectd[27841]: Unhandled python exception in importing module: ImportError: cannot import name DependencyWarning